src/sentry/integrations/notify_disable.py

```diff
@@ -0,0 +1,61 @@
+from typing import Union
+
+from sentry.models import Organization
+from sentry.utils.email import MessageBuilder
+
+provider_types = {
+    "integration": "integrations",
+    "plugin": "plugins",
+    "sentry-app": "developer-settings",
+}
+
+
+def get_url(organization: Organization, provider_type: str, provider: str) -> str:
+    if not provider_type:
+        return str(organization.absolute_url("/settings/integrations/"))
+
+    type_name = provider_types.get(provider_type, "")
```

Comment:
Just to be super safe since the fallback is an empty string here, what if we shuffled the logic around like this: 

```
def get_url(organization: Organization, provider_type: str, provider: str) -> str:
    if provider_type:
        type_name = provider_types.get(provider_type, "")
        if type_name:
            return str(
                organization.absolute_url(
                    f"/settings/{type_name}/{provider}/",
                )
            )

    return str(organization.absolute_url("/settings/integrations/"))
```

This way we account for the possibility of there not being a `provider_type` and `type_name`.